packages/ui-components/style/index.css

```diff
@@ -76,8 +75,6 @@ a:hover {
 }
 
 .jp-InputGroup input:focus {
-  /* box-shadow: inset 0 0 0 var(--jp-border-width) var(--jp-input-active-border-color),
```

Comment:
One more question: is there a reason we aren't using theme colors for this focus highlight, like in the commented code? Any time I see hardcoded colors like this, I wonder how it behaves in different themes.

Reply:
Yes yes, good catch!